styles/index.css

```diff
@@ -40,178 +21,203 @@
         -ms-flex-align: center;
             align-items: center;
     background-color: #cad3d3;
+    text-align: center;
+    max-width: 100%;
 }
 
-.item {
-    width: 362px;
-    height: 768px;  /* nên dùng height: auto */
+.pricing-table, .pricing-table--featured {
     background-color: #ffffff;
     margin: 100px 23px;
     position: relative;
+    width: auto;
+    height: auto;
 }
 
 
 /* ======== IMAGES ======== */
-.image {
+.pricing-table__bgimage {
```

Comment:
nếu nó phụ thuộc vào element cha thì em nên viết như này, áp dụng cả cho những phần còn lại nhé
```suggestion
.price-table .pricing-table__bgimage {
```

Reply:
e đọc trên BEM thì thấy ntn ạ 
![image](https://user-images.githubusercontent.com/8766522/52833138-0d108380-310e-11e9-9e5a-dd8a8bac0275.png)

Họ bảo là hạn chế sự phụ thuộc càng tốt ạ @.@

Reply:
uh thế theo thằng BEM em nhé, anh quen dùng SASS nên viết theo kiểu kia